kibana/templates/deployment.yaml

```diff
@@ -63,6 +63,9 @@ spec:
     {{- end }}
       containers:
       - name: kibana
+        securityContext:
```

Comment:
Same comment as the Elasticsearch PR. It makes the most sense to expose the securityContext to be configurable just like with the pod level securityContext.

https://github.com/elastic/helm-charts/blob/26fe7570fb668b909d11cddb6c5fa2f8cfaed288/kibana/templates/deployment.yaml#L30-L33

Reply:
Sounds good! I've added it as a configurable var. I'll go ahead and do it the same in ES to keep it consistent.